.jenkins/scheduled-tests.Jenkinsfile

```diff
@@ -0,0 +1,234 @@
+/* A Jenkins pipeline that will handle PR and nightly tests
+*  These are the original pipelines:
+*  https://github.com/deislabs/mystikos/blob/main/.azure_pipelines/ci-pipeline-code-coverage-nightly.yml
+*/
+
+pipeline {
```

Comment:
Given that dotnet is no longer an option, I think this will actually be a lot cleaner as there are fewer differences between the nightly and code coverage pipelines